required/tools/multicol.dtx

```diff
@@ -4641,7 +4327,346 @@
 \mc@firstcoltrue
 %    \end{macrocode}
 %  \end{macro}
-% \end{multicols*}
+%
+%
+%
+%
+% \subsection{Using the new mark mechanism}
+%
+% \subsubsection{Helpers}
+%
+% \changes{v2.0a}{2024/11/10}{Using the new mark mechanism}
+%    \begin{macrocode}
+\ExplSyntaxOn
+%<@@=mc>
+%    \end{macrocode}
+%
+%
+%
+%  \begin{macro}{\g_mc_curr_col_int}
```

Comment:
Public? Is this documented somewhere?

Reply:
no, should probably be private (at lease for now)